### PR TITLE
Croatian translation

### PR DESCRIPTION
**Pull request by penzinho**
I am happy to provide Croatian translatiin of this wonderful addon

### CODE REVIEW

**Comment by penzinho** [comment]
WIP

File: resources/language/Croatian/strings.po [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+msgid "Movies"
+msgstr "Filmovi"
+
+msgctxt "#30215"
+msgid "TV Shows"
+msgstr "TV Serije"
+
+msgctxt "#30216"
+msgid "Paste URL"
+msgstr "Zalijepi URL (Paste)
```
Comment (scakemyer) [inline resources/language/Croatian/strings.po]: Missing end quote

**Comment by scakemyer** [comment]
Please rebase/squash those commits, and it'd be great to say which language it is for in the commit message instead of just "Update strings.po" :)

**Comment by penzinho** [comment]
Sorry, first time using github and for it to be even more challenging, it was on my tablet :)

I've fixed all the typos, hopefully my commit message is ok now. I will squash all of the commits during the day.

**Comment by scakemyer** [comment]
Can you rebase as one commit please?

**Comment by scakemyer** [comment]
Merged in c69deea

**Comment by penzinho** [comment]
Sorry I didn't rebase, I wasn't exactly sure how to do it properly. I will try to keep any updates  in one commit.

**Pull request closed without merging**